complete/src/main/java/com/example/restservice/constants/QueueStatus.java

```diff
@@ -0,0 +1,7 @@
+package com.example.restservice.constants;
+
+public enum QueueStatus {
+    ACTIVE,
+    PAUSED,
+    DELETED
```

Comment:
Is deleting going to be a follow up PR?  Let's do it in this PR itself, I feel.

1. We should not allow a paused or active status to be set on a deleted queue.
2. If delete is doing to be a separate request as we had discussed, then we should not allow `PauseQueueRequest` to take deleted as a status in the body.